2_Collections.py

```diff
@@ -0,0 +1,42 @@
+import random  # Import the random module to generate random numbers
```

Comment:
For the next modules please add comments where it's mandatory to describe your code (algorithm or some custom logic). The 95% of the code is self-explanatory. It will safe your time.